### ISSUE TITLE
2.7.3 - Update Trade Building Descriptions to better reflect "Historic Event" reality.

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#9371]
Currently, when an external Trade Route completes under the eyes of a harbor, caravansary, zoo, etc etc this triggers a historic event. However, there is no notification for this or notes in the tooltips, making this a hidden feature easy for a user to not know about.

I have rewritten the building descriptions (the one shown in the civiliopedia as well as in the city dropdown) to better reflect how the buildings function.

**Caravansary / Burial Tomb**
Change this Text: "When a  :C5traderoute: Land Trade Route originating here and targeting another civ is completed, receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output."

To this text: "When a  :C5traderoute: Land Trade Route going from here to another civ completes, a historical event is triggered. Receive a :C5tourism: Tourism boost with the civ based on recent:C5culture: Culture and :C5tourism: Tourism output, and a quarter the boost with all other civs"

**Harbor**
Change this Text: "When a  :C5traderoute: Sea Trade Route originating here and targeting another civ is completed, receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output."

To this Text: "When a  :C5traderoute: Sea Trade Route going from here to another civ completes, a historical event is triggered. Receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output, and a quarter the boost with all other civs"

**Zoo**
Change this Text: "When any  :C5traderoute: Trade Route originating here and targeting another civ is completed, receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output."

To this Text: "When any  :C5traderoute: Trade Route going from here to another civ completes, a historical event is triggered. Receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output, and a quarter the boost with all other civs"

**Bazaar**
Change this Text: "When any  :C5traderoute: Trade Route originating here and targeting another civ is completed, receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output."

To this Text: "When any  :C5traderoute: Trade Route going from here to another civ completes, a historical event is triggered. Receive a :C5tourism: Tourism boost with the civ based on recent :C5culture: Culture and :C5tourism: Tourism output, and a quarter the boost with all other civs"

**Comment by axatin** [comment]
I completely agree with you that this is an inconsistency that should be changed. However, I think it was justified not to call trade route completions "historic events". From a role-play perspective, there is nothing "historical" about completing a trade route, spreading tourism by trade is something very different than gaining tourism influence because of monumental achievements.

I think it would be better to leave the building descriptions as they are and instead change the texts referring to Historic Events.
Arabia UA: "When you gain Instant Tourism with other Civilizations (by triggering a Historic Event or from Building/Policy bonuses to completing a Trade Route), your [ICON_CAPITAL] Capital gains... "
Futurism Policy: "Tourism bonus from Historic Events and Trade Route Completions increased by 20%..."
TXT_KEY_TOP_PANEL_TOURISM_TOOLTIP_CONQUEST_WARNING: "Every [ICON_PUPPET] non-Puppet owned city (excluding [ICON_CAPITAL] Capital) decreases the [ICON_TOURISM] Tourism you produce from Cities, Historic Events and bonuses to Trade Route Completions by {1_Num}% (up to 75%)."

**Comment by Stalker012345** [comment]
> I completely agree with you that this is an inconsistency that should be changed. However, I think it was justified not to call trade route completions "historic events". From a role-play perspective, there is nothing "historical" about completing a trade route, spreading tourism by trade is something very different than gaining tourism influence because of monumental achievements.
> 
> I think it would be better to leave the building descriptions as they are and instead change the texts referring to Historic Events. Arabia UA: "When you gain Instant Tourism with other Civilizations (by triggering a Historic Event or from Building/Policy bonuses to completing a Trade Route), your [ICON_CAPITAL] Capital gains... " Futurism Policy: "Tourism bonus from Historic Events and Trade Route Completions increased by 20%..." TXT_KEY_TOP_PANEL_TOURISM_TOOLTIP_CONQUEST_WARNING: "Every [ICON_PUPPET] non-Puppet owned city (excluding [ICON_CAPITAL] Capital) decreases the [ICON_TOURISM] Tourism you produce from Cities, Historic Events and bonuses to Trade Route Completions by {1_Num}% (up to 75%)."

At the end of the day, trade route completions are historic events by mechanics. They even utilize the same code. This means any thing we do that changes historic events affects these buildings, they are completely intertwined. So I can't agree with your logic here, if we keep them seperate in text, than all that means is that if a new historic event change happens in the future, these texts will be out of sync again.

**Comment by LessRekkless** [comment]
Alternatively, list all of the possible Historic Events in the Civilopedia, including "Trade and Tourism" as one of them.

**Comment by axatin** [comment]
> > I completely agree with you that this is an inconsistency that should be changed. However, I think it was justified not to call trade route completions "historic events". From a role-play perspective, there is nothing "historical" about completing a trade route, spreading tourism by trade is something very different than gaining tourism influence because of monumental achievements.
> > I think it would be better to leave the building descriptions as they are and instead change the texts referring to Historic Events. Arabia UA: "When you gain Instant Tourism with other Civilizations (by triggering a Historic Event or from Building/Policy bonuses to completing a Trade Route), your [ICON_CAPITAL] Capital gains... " Futurism Policy: "Tourism bonus from Historic Events and Trade Route Completions increased by 20%..." TXT_KEY_TOP_PANEL_TOURISM_TOOLTIP_CONQUEST_WARNING: "Every [ICON_PUPPET] non-Puppet owned city (excluding [ICON_CAPITAL] Capital) decreases the [ICON_TOURISM] Tourism you produce from Cities, Historic Events and bonuses to Trade Route Completions by {1_Num}% (up to 75%)."
> 
> At the end of the day, trade route completions are historic events by mechanics. They even utilize the same code. This means any thing we do that changes historic events affects these buildings, they are completely intertwined. So I can't agree with your logic here, if we keep them seperate in text, than all that means is that if a new historic event change happens in the future, these texts will be out of sync again.

this is a good point. Then let's change the text as you proposed
